.github/workflows/build-mingw-on-ubuntu.yml

```diff
@@ -5,6 +5,10 @@ on:
   pull_request:
     branches: [ master ]
 
+concurrency:
+  group: ${{ github.workflow }}-${{ github.head_ref || github.run_id }}
```

Comment:
This is a great feature, but I think this cancelling would only work on  PRs, not on branches since run_id will be unique per run?

Does this actually need to be:

```
group: ${{ github.workflow }}-${{ github.head_ref || gitHub.ref }}
```

Reply:
I think you're right. I guess it depends on what we want. Also I think this would avoid killing manual runs.

Reply:
Regardless, I'll merge and see how it actually behaves!

Reply:
Thanks!